env.example

```diff
@@ -4,6 +4,12 @@ export AZURE_TENANT_ID=
 export AZURE_CLIENT_ID=
 export AZURE_CLIENT_SECRET=
 
+AZURE_MASTER_CLIENT_ID=
```

Comment:
I'm not honestly sure if `export` is actually needed or not but might as well have them all in line

Reply:
i'll make it consistent, unintentional